pytest_django/fixtures.py

```diff
@@ -36,26 +36,39 @@
 def django_db_modify_db_settings_tox_suffix():
     skip_if_no_django()
 
-    tox_environment = os.getenv("TOX_PARALLEL_ENV")
-    if tox_environment:
-        # Put a suffix like _py27-django21 on tox workers
-        _set_suffix_to_test_databases(suffix=tox_environment)
+    return os.getenv("TOX_PARALLEL_ENV")
 
 
 @pytest.fixture(scope="session")
 def django_db_modify_db_settings_xdist_suffix(request):
     skip_if_no_django()
 
-    xdist_suffix = getattr(request.config, "slaveinput", {}).get("slaveid")
-    if xdist_suffix:
+    return getattr(request.config, "slaveinput", {}).get("slaveid")
+
+
+@pytest.fixture(scope="session")
+def django_db_modify_db_settings_suffix(
+    django_db_modify_db_settings_tox_suffix,
+    django_db_modify_db_settings_xdist_suffix
+):
```

Comment:
Why not keep this as `django_db_modify_db_settings_parallel_suffix`?

Reply:
that's... a great point haha. Fixing now